### DISCUSSION TITLE
Static header and footer on each page

### DISCUSSION THREAD

**Discussion by aman2810khan** [wojtekmaj/react-pdf#1602] [General]
I have an HTML string that I am generating dynamically through a json object and I am using a backend service to get the pdf blob for this html string to show the PDF in my browser. Now , I want to add static header and footer on each page. 

The problem I am facing is that I am generating the whole HTML dynamically due to which I won't be able to detect how many pages will be present in my pdf and also cannot locate header and footer of the page.

Can anyone help in how to place this static header and footer on each page ?

**Comment by wojtekmaj** [comment]
It looks like you confused React-PDF with [@react-pdf/renderer](https://github.com/diegomura/react-pdf). Please file an issue in their repo instead.
